mqtt/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# -*- mode: dockerfile -*-
+#
+# An example Dockerfile showing how to build a Rust executable using this
+# image, and deploy it with a tiny Alpine Linux container.
+
+# You can override this `--build-arg BASE_IMAGE=...` to use different
+# version of Rust or OpenSSL.
+ARG BASE_IMAGE=ekidd/rust-musl-builder:beta
```

Comment:
It needs `beta` ? `stable` doesn't work?

Reply:
ah good catch. It was using beta because I started on this before async/await was stabilized.